tests/colr_to_svg.py

```diff
@@ -257,17 +266,24 @@ def _colr_v1_glyph_to_svg(
     glyph_set: ttLib.ttFont._TTGlyphSet,
     view_box: Rect,
     glyph: otTables.BaseGlyphRecord,
+    reuse_cache: ReuseCache,
 ) -> etree.Element:
     glyph_set = ttfont.getGlyphSet()
     svg_root = _svg_root(view_box)
     ascender = ttfont["OS/2"].sTypoAscender
     descender = ttfont["OS/2"].sTypoDescender
     width = ttfont["hmtx"][glyph.BaseGlyph][0]
     font_to_vbox = _map_font_space_to_viewbox(view_box, ascender, descender, width)
-    _colr_v1_paint_to_svg(ttfont, glyph_set, svg_root, font_to_vbox, glyph.Paint)
+    _colr_v1_paint_to_svg(
+        ttfont, glyph_set, svg_root, font_to_vbox, glyph.Paint, reuse_cache
+    )
     return svg_root
 
 
+def _new_reuse_cache() -> ReuseCache:
+    return ReuseCache(0.1, GlyphReuseCache(0.1))
```

Comment:
I thought maybe since it was only for tests and no test currently wants to adjust that it was ok? - lmk if you disagree and I'll fix (potentially in a new PR since I'm inclined to merge)